### PR TITLE
{bio}[GCCcore/8.3.0] GffCompare v0.11.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3416.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/19072425bfc2127418cc1e61534e4982 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/e4b5424d202f0566393fb97b5e7550a8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/0ec71f90e19dc92595cb8734cbccfb9e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/1fd97f761b1064c4fe627d2543f70377 for a full test report.

File: easybuild/easyconfigs/g/GffCompare/GffCompare-0.11.6-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+version = '0.11.6'
+
+homepage = 'https://github.com/gpertea/gffcompare'
+description = """GffCompare provides classification and reference annotation mapping and
+ matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://ccb.jhu.edu/software/stringtie/dl/']
```
Comment (Micket) [inline easybuild/easyconfigs/g/GffCompare/GffCompare-0.11.6-GCCcore-8.3.0.eb:15]:
```suggestion
source_urls = ['https://ccb.jhu.edu/software/stringtie/dl/']
```
despite them redirecting you to http if you go browse the directory, if you specify full path to the source, it seems to actually stick to https, so, i think this works.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/0449c67d69ca6f3325c5733ed528ef42 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**